<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCalibration</class>
 <widget class="QDialog" name="DialogCalibration">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>223</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Calibrate Channel</string>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frameRadioButtons">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioButtonData">
        <property name="text">
         <string>Calibrate from data entries</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonPreset">
        <property name="text">
         <string>Preset calibration</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="labelChannel">
     <property name="text">
      <string>Apply to channel:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="spinBoxChannel">
     <property name="maximum">
      <number>31</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxData">
     <property name="title">
      <string>Data Entries</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableWidgetData">
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <row>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string notr="true">2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>4</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string notr="true">Energy (MeVee)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">Integral</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>0.00</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>2.20</string>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>3.42</string>
         </property>
        </item>
        <item row="3" column="0">
         <property name="text">
          <string>4.20</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBoxPreset">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Preset Calibration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5" columnstretch="1,0">
      <item row="1" column="0">
       <widget class="QLabel" name="labelScale">
        <property name="text">
         <string>Scale (L/MeVee)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditScale">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelOffset">
        <property name="text">
         <string>Offset (L)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditOffset">
        <property name="inputMask">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>labelScale</zorder>
     <zorder>labelOffset</zorder>
     <zorder>lineEditOffset</zorder>
     <zorder>lineEditScale</zorder>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelValidCalib">
     <property name="text">
      <string>Invalid calibration</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioButtonData</tabstop>
  <tabstop>radioButtonPreset</tabstop>
  <tabstop>spinBoxChannel</tabstop>
  <tabstop>tableWidgetData</tabstop>
  <tabstop>lineEditOffset</tabstop>
  <tabstop>lineEditScale</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCalibration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCalibration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonData</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBoxData</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonData</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBoxPreset</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonPreset</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBoxData</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonPreset</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBoxPreset</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidgetData</sender>
   <signal>cellChanged(int,int)</signal>
   <receiver>DialogCalibration</receiver>
   <slot>onDataEntryChanged(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onDataEntryChanged(int,int)</slot>
 </slots>
</ui>
